functions/sendInvoiceStripe/README.md

```diff
@@ -0,0 +1,125 @@
+### How this function works
+
+Use this function to create and send invoices to customers using the Stripe payments platform.
+
+The invoices are automatically customized according to your Stripe settings. After the invoice is sent, you can use the Stripe dashboard to track and get more detailed reports.
+
+As this function uses other Firebase and Google Cloud Platform services, you will be charged if you exceed the service's free tier:
+
+- Cloud Firestore
+- Cloud Functions
+
+### Requirements
+
+- You must have previous knowledge of **Firebase**.
+- You must have previous knowledge of **Firestore**.
+- You must have previous knowledge of **Stripe**.
+
+
+### Function details
+
+To install this function, add the required information to the form with the following parameters:
+
+- **Document Path**: The document path that you'd like this function to listen to. A placeholder should be used for the document ID (e.g., `/invoices/{invoiceId}`).
+- **STRIPE_API_KEY**: The API key to access Stripe services.
+- **DAYS_UNTIL_DUE_DEFAULT**: The default number of days the customer has before the payment due date.
```

Comment:
I think this variable should be DAYS_UNTIL_DUE since I don't see it as a default value and instead it is the explicit amount of day until the invoice is due

Reply:
mm I see below you can overwrite this value

Reply:
ok then it is fine